src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -217,24 +220,22 @@ private boolean logInHelper(GoogleAuthorizationCodeTokenRequest authRequest) {
   }
 
   /**
-   * Logs the user out. Pops up a question dialog asking if the user really
-   * wants to quit.
+   * Logs out all accounts. Pops up a question dialog asking if the user really wants to log out.
    *
-   * @return true if the user logged out, false otherwise
+   * @return true if all accounts were logged out or if no account was logged in; false otherwise
    */
-  public boolean logOut() {
-    return logOut(true);
+  public boolean logOutAll() {
+    return logOutAll(true);
   }
 
   /**
-   * Logs the user out.
+   * Logs out all accounts.
    *
    * @param showPrompt if true, opens a prompt asking if the user really wants
    *          to log out. If false, the user is logged out
-   * @return true if the user was logged out or is already logged out, and false
-   *         if the user chose not to log out
+   * @return true if all accounts were logged out or if no account was logged in; false otherwise
```

Comment:
Is it more useful to say: `return false if the user cancelled the action`?  Otherwise the caller doesn't know if `false` indicates an issue that should be reported somehow.